src/components/ui/Alert.jsx

```diff
@@ -28,6 +29,11 @@ const Alert = ({
     <div className="alert-content">
       <span role="alert">{description}</span>
     </div>
+    <div className="alert-link">
+      <a onClick={onClose}>
```

Comment:
If there is no href this isn't a link (and it won't display a pointer cursor).
However, this looks exactly like a `<Button variant="tertiary">`.